### PR TITLE
Bug 2046594: "Requested template could not be found" while creating VM from user-created template

### PR DESCRIPTION
**Pull request by avivtur**
**Fixes**:
https://bugzilla.redhat.com/show_bug.cgi?id=2046594

**Analysis / Root cause**:
When 'All projects' is selected, we insert 'default' namespace as a default, and trying to reach template's resources on 'default` namespace instead of the template's actual namespace

**Solution Description**:
changing the namespace when getting template's resources to the correct namespace

**Screen shots / Gifs for design review**:

**Before**:

https://user-images.githubusercontent.com/67270715/151383327-23159c6f-4921-4772-9fe4-9c70c90948cb.mp4

**After**:

https://user-images.githubusercontent.com/67270715/151383351-378375ce-3ba4-44da-8eaf-66a526b073cd.mp4

Signed-off-by: Aviv Turgeman <aturgema@redhat.com>

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/src/components/create-vm/create-vm.tsx [hunk 1]
```diff
@@ -176,6 +176,7 @@ export const CreateVM: React.FC<RouteComponentProps<{ ns: string }>> = ({ match,
   const { t } = useTranslation();
   const searchParams = new URLSearchParams(location && location.search);
   const initData = parseVMWizardInitialData(searchParams);
+  const templateNS = initData.commonTemplateName ? 'openshift' : initData.userTemplateNs;
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/src/components/create-vm/create-vm.tsx:179]: nitpick: We should have `openshift` stored as a constant somewhere.

**Review by pcbailey — COMMENTED** [review]
One nitpick, but definitely not worth holding up the PR for.
/lgtm

**Comment by gouyang** [comment]
Watched the "after" video, the template ns is "test", but the wizard is using"default"?

**Pull request merged**